src/styles/blocks/seconds.scss

```diff
@@ -0,0 +1,12 @@
+.seconds {
+  position: absolute;
+  top: 72.5px;
+  left: 247.5px;
```

Comment:
Ok, what will be in case when customer will decide to change clock size, you will recalculate these parameters manually?

Reply:
Fixed, I suppose...